docs/usage-as-end-user.md

```diff
@@ -92,6 +92,9 @@ spec:
       volume:
         size: 50Gi
         type: storage_1
+      zones: # Optional list of facilities; if not provided then a random facility within the metro will be chosen.
```

Comment:
Suggest: 

> # Optional list of facilities, all of which MUST be in the metro; if not provided, then random facilities within the metro will be chosen for each machine.

Reply:
Ouch, sorry about the formatting

Reply:
Done! However, this will also be technically ensured by validation performed by the `gardener-apiserver` (see `CloudProfile`; the operator configures metros + facilities here and users can only choose their values from this list).